f/fftw/manifest.x86_64.jsonc

```diff
@@ -0,0 +1,81 @@
+/** Human readable report. This is not consumed by boulder */
+{
+	"manifest-version": "0.2",
+	"packages": {
+		"fftw": {
+			"build-depends": [
+				"binary(makeinfo)"
+			],
+			"depends": [
+				"interpreter(/usr/lib/ld-linux-x86-64.so.2(x86_64))",
+				"soname(libc.so.6(x86_64))",
+				"soname(libm.so.6(x86_64))"
+			],
+			"files": [
+				"/usr/bin/fftw-wisdom",
+				"/usr/bin/fftw-wisdom-to-conf",
+				"/usr/bin/fftwf-wisdom",
+				"/usr/lib/libfftw3.so.3",
+				"/usr/lib/libfftw3.so.3.6.10",
+				"/usr/lib/libfftw3_threads.so.3",
+				"/usr/lib/libfftw3_threads.so.3.6.10",
+				"/usr/lib/libfftw3f.so.3",
+				"/usr/lib/libfftw3f.so.3.6.10",
+				"/usr/lib/libfftw3f_threads.so.3",
+				"/usr/lib/libfftw3f_threads.so.3.6.10",
+				"/usr/share/info/dir",
```

Comment:
Oh, this directory should be deleted too.

Reply:
Yea I noticed that after I decided to push the fixed build sigh